test/runtime/Policies.go

```diff
@@ -951,22 +952,40 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 	Context("TestsEgressToHost", func() {
 		hostDockerContainer := "hostDockerContainer"
 		hostIP := "10.0.2.15"
+		otherHostIP := ""
 
 		BeforeAll(func() {
 			By("Starting httpd server using host networking")
 			res := vm.ContainerCreate(hostDockerContainer, helpers.HttpdImage, helpers.HostDockerNetwork, "-l id.hostDockerContainer")
 			res.ExpectSuccess("unable to start Docker container with host networking")
+
+			By("Detecting host IP in world CIDR")
+			res = vm.Exec("ip route | grep '192.168' | head -n 1")
+			res.ExpectSuccess("unable to detect host IP for CIDR test")
+			routeParams := strings.Split(res.Output().String(), " ")
+			for _, param := range routeParams {
+				if net.ParseIP(param) != nil {
+					otherHostIP = param
+					break
+				}
+			}
+			Expect(otherHostIP).To(Not(Equal("")))
+			By(fmt.Sprintf("Using %s for world CIDR IP", otherHostIP))
 		})
 
 		AfterAll(func() {
 			vm.ContainerRm(hostDockerContainer)
 		})
 
 		BeforeEach(func() {
-			By(fmt.Sprintf("Pinging %s from %s before importing policy (should work)", api.EntityHost, helpers.App1))
+			By(fmt.Sprintf("Pinging %s from %s before importing policy (should work)", hostIP, helpers.App1))
```

Comment:
On `By` messages I would suggest the use of `%q` so the variables are into `''` and it's easy to understand what are the moving parts ;-)

Reply:
I assume it's always the same for strings, but for objects this is better? OK :)